src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -524,9 +524,11 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
      * If the system property is not defined, and the {@link java.security.Security} property
      * {@code jdk.serialFilter} is defined then it is used to configure the filter.
      * The filter is created as if {@link #createFilter(String) createFilter} is called;
-     * if the filter string is invalid, an {@link ExceptionInInitializerError} is thrown.
-     * Otherwise, the filter is not configured during initialization and
-     * can be set with {@link #setSerialFilter(ObjectInputFilter) Config.setSerialFilter}.
+     * if the filter string is invalid, an {@link ExceptionInInitializerError} is thrown
+     * and the initialization fails; subsequent attempts to use the configuration or
+     * serialization will fail with an implementation specific exception.
+     * If the system property {@code jdk.serialFilter} is not set on the command line
```

Comment:
Hello Roger,
Thank you for rearranging these lines. It reads much more clearly. One tiny final question - this new line now states `If the system property {@code jdk.serialFilter} is not set on the command line it can be set with ....`. However, this property if not set on the command line could have instead been set as a `java.security.Security` property (in a file). The javadoc does mention this a few lines back. So do you think this new line should be reworded to something like `If the filter is neither set as a system property on the command line nor as a security property then it can be set with...`